docs/csharp/discards.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Discards - C# Guide| Microsoft Docs
+description: Describes C#'s support for discards, which are unassigned, discardable variables, and the ways in which discards can be used.
+keywords: .NET,.NET Core
+author: rpetrusha
+ms.author: ronpet
+ms.date: 07/21/2017
+ms.topic: article
+ms.prod: .net
+ms.technology: devlang-csharp
+ms.devlang: csharp
+---
+# Discards - C# Guide
+
+Starting with C# 7, C# supports discards, which are temporary, dummy variables that are intentionally unused in application code. Discards are equivalent to unassigned variables; they do not have a value. Discards can reduce memory allocations and and enhance the readability and maintainability of your code.
+
+You indicate that a variable is a discard by assigning it the underscore (`_`) as its name. For example, the following method call returns a 3-tuple in which the first and second values are discards:
+
+```csharp
+(var -, -, area) = city.GetCityInformation(cityName);
+```
+
+In C# 7, discards are supported in assignments in the following contexts:
+
+- Tuple and object [deconstruction](deconstruct.md).
+- Pattern matching with [is](language-reference/keywords/is.md) and [switch](language-reference/keywords/switch.md).
+- Calls to methods with `out` parameters.
+- A standalone `_` when no `_` is in scope.
+
+When `_` is a valid discard, attempting to retrieve its value or use it in an assignment operation generates compiler error CS0301, "The name '_' does not exist in the current context". 
```

Comment:
This is a key point, and maybe could be expounded upon more: The "_" is not assigned a value, it may not even be given storage. If it were an actual variable, you could not discard more than one member, as you do in your example ablve.